doc/ref/configuration/minion.rst

```diff
@@ -2713,6 +2713,34 @@ the ``extra_minion_data`` parameter will be
 
     {"opt1": "value1", "opt2": {"subopt1": "value2"}}
 
+``ssh_merge_pillar``
+--------------------
+
+.. versionadded:: 2018.3.2
+
+Default: ``True``
+
+Merges the compiled pillar data with the pillar data already available globally.
+This is useful when using ``salt-ssh`` or ``salt-call --local`` and overriding the pillar
+data in a state file:
+
+.. code-block:: yaml
+
+    apply_showpillar:
+      module.run:
+        - name: state.apply
+        - mods:
+          - showpillar
+        - kwargs:
+              pillar:
+                  test: "foo bar"
+
+If set to ``True`` the ``showpillar`` state will have access to the
+global pillar data.
+
+If set to ``False`` only the overriding pillar data will be available
+to the ``showpillar`` state.
+
```

Comment:
Same grammar point here as well: If set to ``False``, only the overriding pillar data will be available...